onboarding_project/onboarding_project/settings.py

```diff
@@ -25,8 +25,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+LOGIN_URL = '/admin/login/'
```

Comment:
Because `/admin/metric` page was accessed even if the user wasn't logged in.
So I marked that `view` as `login_required`. 
So what `login_required` do is that it redirects to `LOGIN_URL` if the page is accessed without login.
Default `LOGIN_URL` for django is `/accounts/login/`. So I had to change it to `/admin/login/'